study-builder/studies/shared/family-history/family-history-aunt-uncle.conf

```
@@ -0,0 +1,178 @@
+{
+  include required("../../snippets/activity-general-form.conf"),
+  "studyGuid": ${id.study},
+  "parentActivityCode": "FAMILY_HISTORY",
+  STABLE_ID_PREFIX="AUNT_UNCLE"
+  "activityCode": "FAMILY_HISTORY_AUNT_UNCLE",
+  "canDeleteInstances": true,
+  "displayOrder": 5,
+  "versionTag": "v1",
+  "allowUnauthenticated": false,
+  "listStyleHint": "NONE",
+  "CONDITIONAL_CANCER_LIST_BLOCK": {
+    include required("snippets/family-history-block-conditional-cancer-list.conf")
+  }
+  "translatedNames": [
+    {
+      "language": "en",
+      "text": """$ddp.answer("AUNT_UNCLE_RELATION", "Aunt or Uncle")"""
+    }
+  ],
+  "translatedTitles": [
+    {
+      "language": "en",
+      "text": "Aunts & Uncles"
```

According to the mockup, this should be singular since they're only filling out one relative in the popup modal.
```suggestion
      "text": "Aunt or Uncle"
```